watertap/examples/case_studies/santa_barbara/santa_barbara.py

```diff
@@ -0,0 +1,443 @@
+
+    # chlorination
+    m.db.get_unit_operation_parameters("chlorination")
+    prtrt.chlorination.load_parameters_from_database(use_default_removal=True)
+
+    # static mixer
+    m.db.get_unit_operation_parameters("static_mixer")
+    prtrt.static_mixer.load_parameters_from_database(use_default_removal=True)
+
+    # storage tank
+    m.db.get_unit_operation_parameters("storage_tank")
+    prtrt.storage_tank_1.load_parameters_from_database(use_default_removal=True)
+    prtrt.storage_tank_1.storage_time.fix(2)
+
+    # media filtration
+    m.db.get_unit_operation_parameters("media_filtration")
+    prtrt.media_filtration.load_parameters_from_database(use_default_removal=True)
+
+    # backwash handling
+    m.db.get_unit_operation_parameters("backwash_solids_handling")
+    prtrt.backwash_handling.load_parameters_from_database(use_default_removal=True)
+
+    # anti-scalant
+    prtrt.anti_scalant_addition.load_parameters_from_database()
+    prtrt.anti_scalant_addition.chemical_dosage.fix(5)
+
+    # cartridge filtration
+    m.db.get_unit_operation_parameters("cartridge_filtration")
+    prtrt.cartridge_filtration.load_parameters_from_database(use_default_removal=True)
+
+    # ---desalination---
+    # splitter (no degrees of freedom)
+
+    # pump 1, high pressure pump, 2 degrees of freedom (efficiency and outlet pressure)
+    desal.P1.efficiency_pump.fix(0.80)  # pump efficiency [-]
+    operating_pressure = 70e5 * pyunits.Pa
+    desal.P1.control_volume.properties_out[0].pressure.fix(operating_pressure)
+
+    # pressure exchanger
+    desal.PXR.efficiency_pressure_exchanger.fix(0.95)  # pressure exchanger efficiency [-]
+
+    # pump 2, booster pump, 1 degree of freedom (efficiency, pressure must match high pressure pump)
+    desal.P2.efficiency_pump.fix(0.80)
+
+    # mixer, no degrees of freedom
+
+    # RO unit
+    desal.RO.A_comp.fix(4.2e-12)  # membrane water permeability coefficient [m/s-Pa]
+    desal.RO.B_comp.fix(3.5e-8)  # membrane salt permeability coefficient [m/s]
+    desal.RO.channel_height.fix(1e-3)  # channel height in membrane stage [m]
+    desal.RO.spacer_porosity.fix(0.97)  # spacer porosity in membrane stage [-]
+    desal.RO.permeate.pressure[0].fix(101325)  # atmospheric pressure [Pa]
+    desal.RO.width.fix(1000)  # stage width [m]
+    desal.RO.area.fix(flow_vol * 4.5e4 * pyunits.s/pyunits.m)  # stage area [m2] TODO: replace with actual area
+
+    # ---posttreatment---
+    # storage tank 2
+    psttrt.storage_tank_2.load_parameters_from_database(use_default_removal=True)
+    psttrt.storage_tank_2.storage_time.fix(1)
+
+    # uv aop
+    m.db.get_unit_operation_parameters("uv_aop")
+    psttrt.uv_aop.load_parameters_from_database(use_default_removal=True)
+    psttrt.uv_aop.uv_reduced_equivalent_dose.fix(350)  # TODO: check this was the right thing to fix
+    psttrt.uv_aop.uv_transmittance_in.fix(0.95)  # TODO: check this was the right thing to fix
```

Comment:
These look like the right things to fix. And by loading parameters, I suppose you also fixed `oxidant_dose` to its default value of 5 mg/L which matches what I see in `treatment_train_setup.csv`